src/indentation.rs

```diff
@@ -59,6 +61,33 @@ pub fn indent(s: &str, prefix: &str) -> String {
     result
 }
 
+/// To add a specific indent to just the first line, and then the same length
+/// of spaces to the remaining lines. Whitespace does not count as the first line.
+///
+/// ```
+/// use textwrap::bullet;
+///
+/// assert_eq!(bullet("The first line\nand some text\ngiving more details.", "  - "),
+///            "  - The first line\n    and some text\n    giving more details.\n");
+/// ```              
+pub fn bullet(s: &str, prefix: &str) -> String {
+    let mut result = String::new();
+    let prefix_len = UnicodeWidthStr::width(prefix);
+    let spaces = format!("{:w$}", "", w = prefix_len);
```

Comment:
I think this can be written easier as
```
let spaces = " ".repeat(prefix.width());
```
The `prefix_len` variable is then not needed (the `UnicodeWidthStr` trait adds a `width` method to `str`).